client/layout/sidebar/item.jsx

```diff
@@ -76,6 +76,13 @@ export default function SidebarItem( props ) {
 							: props.label
 					}
 					{ !! count && <Count count={ count } /> }
+					{ !! badge && (
+						<span className="sidebar__menu-link-badge">
+							{ 'string' === typeof props.badge
+								? stripHTML( decodeEntities( props.badge ) )
+								: props.badge }
```

Comment:
Is the `badge` ever anything else but string or null?

Reply:
Not at the moment, removed the unncessary logic.